tapdb/assets_store.go

```diff
@@ -294,7 +293,8 @@ type ActiveAssetsStore interface {
 		assetID []byte) (sqlc.FetchAssetMetaForAssetRow, error)
 }
 
-type InsertRecvProofTxAttemptParams = sqlc.InsertReceiverProofTransferAttemptParams
+type LogProofTransferAttemptParams = sqlc.LogProofTransferAttemptParams
```

Comment:
Style nit: can nest under the same `type` statement. Also might as well make a shorter name, as otherwise only saving on the the `sqlc` qualifier.